classes/author.rb

```diff
@@ -10,4 +11,9 @@ def initialize(firstname, lastname)
     @lastname = lastname
     @items = []
   end
+
+  def add_item(item)
+    @items << item unless @items.include?(item)
+    item.author = self
+  end
```

Comment:
- Addressed ✔

Reply:
Hi @Yazino12 thanks for your guidance and kind feedback.